_includes/tutorials/versioned-ktables/kstreams/markup/test/make-topology-test.adoc

```diff
@@ -0,0 +1,18 @@
+////
```

Comment:
delete this comment and the one below? Or do we leave these in our tutorials?

Reply:
These are relics from when we expected to get lots of external contributions.  I've deleted them both.